src/options/optionsMain.ts

```diff
@@ -130,6 +132,26 @@ export async function getOptions(rawOptions: any): Promise<AppOptions> {
     }
   }
 
+  if (rawOptions.wvvmp) {
+    process.env.ELECTRON_MIRROR =
+      'https://github.com/castlabs/electron-releases/releases/download/';
+    const electronVersion = options.packager.electronVersion;
+    options.packager.electronVersion = `${electronVersion}-wvvmp`;
+    try {
+      await axios.get(
+        `https://github.com/castlabs/electron-releases/releases/tag/v${electronVersion}-wvvmp`,
+      );
+    } catch (error) {
+      throw `\nERROR: castLabs Electron version "${options.packager.electronVersion}" does not exist. \nVerify versions at https://github.com/castlabs/electron-releases/releases. \nAborting.`;
+    }
+    log.warn(
+      `\nATTENTION: Using the **unofficial** Electron for Content Security from castLabs`,
+      "\nIt implements Google's Widevine Content Decryption Module (CDM) for DRM-enabled playback.",
+      `\nBe sure to select a valid version from https://github.com/castlabs/electron-releases/releases !`,
+      `\nSimply abort & re-run without passing the wvvmp flag to default to ${DEFAULT_ELECTRON_VERSION}`,
+    );
```

Comment:
Nits:

1. Use an extra variable to clarify between electron version vs widevine-prefixed electron version
2. Use an empty line to separate the early exit from the actual work
3. `process.env.ELECTRON_MIRROR` should be set _after_ the early exit
4. No need to repeat `Be sure to select a valid version` in the log.warn, at this point we validated the requested version exists 🙂

```suggestion
    const widevineElectronVersion = `${options.packager.electronVersion}-wvvmp`;
    try {
      await axios.get(
        `https://github.com/castlabs/electron-releases/releases/tag/v${widevineElectronVersion}`,
      );
    } catch (error) {
      throw `\nERROR: castLabs Electron version "${widevineElectronVersion}" does not exist. \nVerify versions at https://github.com/castlabs/electron-releases/releases. \nAborting.`;
    }

    options.packager.electronVersion = widevineElectronVersion;
    process.env.ELECTRON_MIRROR =
      'https://github.com/castlabs/electron-releases/releases/download/';
    log.warn(
      `\nATTENTION: Using the **unofficial** Electron from castLabs`,
      "\nIt implements Google's Widevine Content Decryption Module (CDM) for DRM-enabled playback.",
      `\nSimply abort & re-run without passing the wvvmp flag to default to ${DEFAULT_ELECTRON_VERSION}`,
    );
```